src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
The `getRandomBall` method is used as a static method, which is not recommended. You should create an instance of the Lottery class and call the method on that instance instead.